eqs/Cargo.toml

```diff
@@ -27,7 +27,7 @@ net = { git = "https://github.com/EspressoSystems/net.git" }
 rand = "0.8.4"
 rand_chacha = { version = "0.3.1", features = ["serde1"] }
 reef = { git = "https://github.com/EspressoSystems/reef.git" }
-seahorse = { git = "https://github.com/EspressoSystems/seahorse.git", features = ["testing"] }
+seahorse = { git = "https://github.com/EspressoSystems/seahorse.git", rev = "1acd9798408151e6e674dbf59af425bb5cfd8f58", features = ["testing"] }
```

Comment:
I think we've decided it's better to leave the `rev` out of `Cargo.toml` and just update it in `Cargo.lock` (i.e. `cargo update -p seahorse`)